warehouse/integrations/deltalake-native/deltalake.go

```diff
@@ -905,6 +905,8 @@ func (d *Deltalake) partitionQuery(ctx context.Context, tableName string) (strin
 	}
 	defer func() { _ = rows.Close() }()
 
+	_ = rows.Err() // ignore error
```

Comment:
We need to read Columns, but not the rows, no need to check for errors.

I prefer using `_ = rows.Err()`, instead of `//nolint:rowserrcheck` because it works across all linters.